nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
```

Comment:
(I didn't want to use `internal/log/` and possibly conflict with the stdlib in files that import it, but maybe that's not really a problem in practice)